### PR TITLE
[RHACS] [Docs] [rhacs-docs-4.6] ROX-27309: Release notes for 4.6.1

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.6

Issue:
https://issues.redhat.com/browse/ROX-27309

Link to docs preview: https://86217--ocpdocs-pr.netlify.app/openshift-acs/latest/release_notes/46-release-notes#about-release-461_release-notes-46

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-4.6`

### CODE REVIEW

File: release_notes/46-release-notes.adoc [hunk 1]
```diff
@@ -365,4 +366,15 @@ With this release, the response includes the new `code` field that includes the
...
+[id="about-release-461_{context}"]
+== About release 4.6.1
+
+*Release date*: 16 December 2024
+
+This release of {product-title-short} fixes the following bugs:
+
+* Fixed an issue where Sensor went online prematurely due to HTTP reachability, assuming the gRPC connection was active.
+
+* Fixed an issue where the HTML code in column values was rendered in PDFs due to insufficient sanitization.
```
Comment (lahinson) [inline release_notes/46-release-notes.adoc:378]:
```suggestion
* The HTML code in column values was rendered in PDFs due to insufficient sanitization.
```

**Review by lahinson — COMMENTED** [review]
Looks good! Just one suggestion for you.

File: release_notes/46-release-notes.adoc [hunk 2]
```diff
@@ -365,4 +366,15 @@ With this release, the response includes the new `code` field that includes the
...
 
-include::modules/image-versions.adoc[leveloffset=+1]
+[id="about-release-461_{context}"]
+== About release 4.6.1
+
+*Release date*: 16 December 2024
+
+This release of {product-title-short} fixes the following bugs:
+
+* Fixed an issue where Sensor went online prematurely due to HTTP reachability, assuming the gRPC connection was active.
```
[the remark this answers is not in the document]
Comment (agantony) [reply release_notes/46-release-notes.adoc:376]: Thank you for the suggestion, but I would prefer to leave it as it is to maintain consistency with other versions.
https://docs.openshift.com/acs/4.5/release_notes/45-release-notes.html#about-release-452_release-notes-45

[the remark this answers is not in the document]
Comment (lahinson) [reply release_notes/46-release-notes.adoc:376]: Understood 👍

**Comment by kcarmichael08** [comment]
Opened #86418 to change date.

**Pull request closed without merging**